SECURITY.md

```diff
@@ -4,7 +4,9 @@ Here potential security and privacy risks are documented so that they can be tra
 
 ## API Keys in Source Code
 
-Currently, we are using `react-native-dotenv` for environment variables. These environment variables may include API keys. Because this is an app not simply intended to run a remote server where only a few users will be able to access the source code, but instead directly on a user's device a user can decompile the source code in order to determine our API keys in plaintext. If a user does this, they will gain unauthorized access to our API key, potentially exhausting our quotas causing functionality to go offline, or charging us money.
+~~Currently, we are using `react-native-dotenv` for environment variables. These environment variables may include API keys. Because this is an app not simply intended to run a remote server where only a few users will be able to access the source code, but instead directly on a user's device a user can decompile the source code in order to determine our API keys in plaintext. If a user does this, they will gain unauthorized access to our API key, potentially exhausting our quotas causing functionality to go offline, or charging us money.~~
+
+As of #90, `react-native-dotenv` has been removed.
```

Comment:
Can we just remove this section? No reason to explain it all when we immediately say "we're not using it anymore"

If you want, we could replace it with an explanation of the env file ("we're using a module to hold all of our env variables because they aren't sensitive so it's okay to commit them")

Reply:
Perhaps it's useful to keep it so we can show Vesonder for the next demo what the issue was and how we solved it. I do agree though that the new env file should have an explanation.

Reply:
> Perhaps it's useful to keep it so we can show Vesonder

yeah, that was the main reason I kept it around, so that we have documentation of the problem existing and being solved. its also struck through with markdown so when you view it through Github you can clearly tell its not relevant anymore

Reply:
ok cool